docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -4,154 +4,203 @@ approvers:
 title: Schedule GPUs
 ---
 
```

Comment:
Can we mark this as alpha ?
i.e: `+{% include feature-state-alpha.md %}`

Reply:
This is describing a task ("how to use GPUs") and not a particular feature. The device plugin page linked from includes this already. In this page, we have mentioned multiple times that this is alpha etc.

Reply:
Sounds good